internal/runfiles/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('build_bazel_rules_nodejs/internal/linker').runfiles;
```

Comment:
feels like this is wishing we had already found a way to split up the linker code and move the runfiles helper here.
If we tried to do that, do we still run into the bootstrap problem of needing runfiles available before we've linked it to a place we can load from?